### PR TITLE
Create crunchyroll and funimation

### PR DESCRIPTION
**Pull request by Shelling0**
These are two mainstream anime steaming sites.

### CODE REVIEW

**Comment by IceCodeNew** [comment]
Can you put these domains into some categories?

**Comment by Shelling0** [comment]
> Can you put these domains into some categories?

Crunchyroll and Funimation have been lumped together into **anime**, and more streaming sites have been added. 

Note that gogoanime, another anime site, has been added by someone else previously, and still remains a separate file. You may want to put gogoanime into **anime** as well.

**Comment by IceCodeNew** [comment]
> > Can you put these domains into some categories?
> 
> Crunchyroll and Funimation have been lumped together into **anime**, and more streaming sites have been added.
> 
> Note that gogoanime, another anime site, has been added by someone else previously, and still remains a separate file. You may want to put gogoanime into **anime** as well.


Yes, I definitely want `gogoanime` to be put into the `anime` category.
Could you please accomplish that in this PR?

File: data/anime [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# 9amine
+9anime.to
+9anime.id
+9anime.cz
+9anime.ws
+
+#animedao
```
Comment (IceCodeNew) [inline data/anime:7]:
```suggestion
# animedao
```

File: data/anime [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# 9amine
+9anime.to
+9anime.id
```
Comment (IceCodeNew) [inline data/anime:3]: Please sort these domains.

**Review by IceCodeNew — CHANGES_REQUESTED** [review]

**Comment by Shelling0** [comment]
> > > Can you put these domains into some categories?
> > 
> > 
> > Crunchyroll and Funimation have been lumped together into **anime**, and more streaming sites have been added.
> > Note that gogoanime, another anime site, has been added by someone else previously, and still remains a separate file. You may want to put gogoanime into **anime** as well.
> 
> Yes, I definitely want `gogoanime` to be put into the `anime` category. Could you please accomplish that in this PR?

Sure.

**Review by IceCodeNew — APPROVED** [review]

**Review by IceCodeNew — COMMENTED** [review]
I am ok with these changes.
Only concerned about the deletion of the gogoanime file as this move could impact our users.

**Comment by IceCodeNew** [comment]
Seems this PR is not getting care for about two weeks. I will merge this PR.
Anyway, we can do another PR to fix the problem should this PR cause.

**Comment by IceCodeNew** [comment]
Thanks for your contribution!

**Pull request merged**